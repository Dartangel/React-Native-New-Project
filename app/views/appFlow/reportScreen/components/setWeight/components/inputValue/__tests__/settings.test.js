import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import InputValue from '../index';

describe('InputValue', () => {
    test('InputValue snapshot', () => {
        const renderer = new ShallowRenderer();
        const snapshot = renderer.render(<InputValue />);
        expect(snapshot).toMatchSnapshot();
    });
});