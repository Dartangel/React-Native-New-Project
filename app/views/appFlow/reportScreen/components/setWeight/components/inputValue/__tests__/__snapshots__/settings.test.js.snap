// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InputValue InputValue snapshot 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 0.4,
      "justifyContent": "center",
    }
  }
>
  <Text
    style={
      Object {
        "fontSize": 25,
      }
    }
  />
  <View
    style={
      Object {
        "alignItems": "flex-end",
        "flex": 1,
        "flexDirection": "row",
        "justifyContent": "center",
      }
    }
  >
    <ForwardRef
      style={
        Object {
          "backgroundColor": "transparent",
          "justifyContent": "flex-end",
          "width": "70%",
        }
      }
      underlineColor="grey"
    />
    <TouchableOpacity
      activeOpacity={0.2}
      delayLongPress={600}
      extraButtonProps={
        Object {
          "rippleColor": "transparent",
        }
      }
      style={
        Object {
          "backgroundColor": "#08DBA5",
          "borderColor": "grey",
          "borderRadius": 4,
          "borderWidth": 1,
          "justifyContent": "flex-end",
          "margin": 5,
          "padding": 5,
        }
      }
    >
      <Text>
        Kg
      </Text>
    </TouchableOpacity>
    <TouchableOpacity
      activeOpacity={0.2}
      delayLongPress={600}
      extraButtonProps={
        Object {
          "rippleColor": "transparent",
        }
      }
      style={
        Object {
          "backgroundColor": "#08DBA5",
          "borderColor": "grey",
          "borderRadius": 4,
          "borderWidth": 1,
          "justifyContent": "flex-end",
          "margin": 5,
          "padding": 5,
        }
      }
    >
      <Text>
        lb
      </Text>
    </TouchableOpacity>
  </View>
</View>
`;
